src/mem/cache/prefetch/Prefetcher.py

```diff
@@ -192,6 +192,7 @@ class StridePrefetcher(QueuedPrefetcher):
     use_requestor_id = Param.Bool(True, "Use requestor id based history")
 
     degree = Param.Int(4, "Number of prefetches to generate")
+    distance = Param.Int(0, "How far ahead of the demand stream to prefetch")
```

Comment:
Could you add a sentence here about what "0" means? Does 0 mean the *next* block or the current block?

If "0" means the current block, then we should probably set the default to "1", right?